### ISSUE TITLE
Precompiled Binaries

### ISSUE THREAD

**Issue by atrope** [Tienisto/rhttp#20]
Hi there!  
How can we enable `precompiled_binaries` so that we don't need to build from scratch every time locally? I also noticed you've set it to always rebuild in release mode, but it should be possible to use precompiled binaries to speed up the testing process.  

Thanks!

![image](https://github.com/user-attachments/assets/471eda75-d287-44ae-8c65-fbfcaa6f1842)

**Comment by felipecastrosales** [comment]
hey @Tienisto I'm facing this same problem. can you see this or suggest anything?

**Comment by Tienisto** [comment]
I don't think that compiling in debug mode provides a significant reduction in compile time. I've changed it to release mode because it was slow in debug mode compared to Dart.

I am thinking of providing binaries to download but I don't have a solution yet.

**Comment by atrope** [comment]
@Tienisto can't we have a way to use already precompiled binaries? So the first time it would build but if it exists  we could use them instead of compiling each build

**Comment by Tienisto** [comment]
It should already cache the binaries between builds as far as I've seen

**Comment by Tienisto** [comment]
I don't have plans to implement this because I don't work on any project that uses rhttp right now.
You are free to add the github action for this that uploads the binary as github release asset.
